src/containers/chat/New_Chat/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
what is the difference between New_Chat and Existing_chat?
Surely this should be a single `Chat` component with options?